doc/governance/accept_recovery.rst

```diff
@@ -17,7 +17,10 @@ A member proposes to recover the network and other members can vote on the propo
         "actions": [
             {
                 "name": "transition_service_to_open",
-                "args": null
+                "args": {
+                    "previous_service_identity": "-----BEGIN CERTIFICATE-----\nMIIBuDCCAT6gAwIBAgIRANWm4xJICc6i4sir+jRXE2gwCgYIKoZIzj0EAwMwFjEU\nMBIGA1UEAwwLQ0NGIE5ldHdvcmswHhcNMjIwMzExMTcwNTEyWhcNMjIwMzEyMTcw\nNTExWjAWMRQwEgYDVQQDDAtDQ0YgTmV0d29yazB2MBAGByqGSM49AgEGBSuBBAAi\nA2IABOyCL4ZOG0mu7fLpciVWcDHFp1dOVr1osONVgG/fhjjZryR/HS5xIc20d96L\nN4yl6qbtoEGE1r1juQB44xoEKOox7OLRD2S0N1/T/DfdCIdgyv5rAVIFCMZVtxGA\nsg6I26NQME4wDAYDVR0TBAUwAwEB/zAdBgNVHQ4EFgQUAz9Pfzi3sEN2w6KuncV2\n0wFXDC0wHwYDVR0jBBgwFoAUAz9Pfzi3sEN2w6KuncV20wFXDC0wCgYIKoZIzj0E\nAwMDaAAwZQIxAKM+T5Lvv4/2nKn8ZL87DkKiBwaGh1kLmrM/0xLhlQYgRp13iqw8\ndt/Zm+/dLCZe/AIwBrgsP5YM2TZ/AAHgC50H8+DKd0k/DfVIy28qhMb/6jr1bCMp\nf0CN7wvG22F59hDa\n-----END CERTIFICATE-----\n\u0000",
```

Comment:
Our other cert-pem-in-proposal instances don't include the trailing null byte. Is that required for this identity? If not, please remove it for consistency.

Reply:
That's a good question! It shouldn't be required (post mbedTLS), but I'll run some tests.

Reply:
Yep, all fine without the null bytes. I added a few lines of code to actively remove them when reading a certificate file, but I'm not sure we should keep that, because it looks more complicated than just not worrying about them. They are in the last commit and I'm happy to revert that if you agree.